src/components/forms/ProductSelect/ProductSelect.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+
+import AsyncSelectField from '../SelectField/AsyncSelectField';
```

Comment:
```suggestion
import {AsyncSelectField} from 'components/forms';
```
this is possible with the version in `main` now :)